integration-tests/deployment/ccip/test_helpers.go

```diff
@@ -232,3 +248,68 @@ func AddLanesForAll(e deployment.Environment, state CCIPOnChainState) error {
 	}
 	return nil
 }
+
+const (
+	// nolint:lll
+	// https://github.com/smartcontractkit/chainlink/blob/a348b98e90527520049c580000a86fb8ceff7fa7/contracts/src/v0.8/tests/MockV3Aggregator.sol#L76-L76
+	MockLinkAggregatorDescription = "v0.8/tests/MockV3Aggregator.sol"
+	// MockWETHAggregatorDescription WETH use description from MockETHUSDAggregator.sol
+	// nolint:lll
+	// https://github.com/smartcontractkit/chainlink/blob/a348b98e90527520049c580000a86fb8ceff7fa7/contracts/src/v0.8/automation/testhelpers/MockETHUSDAggregator.sol#L19-L19
+	MockWETHAggregatorDescription = "MockETHUSDAggregator"
+)
+
+var (
+	MockLinkPrice = big.NewInt(5e18)
+	// MockDescriptionToTokenDescriptor maps a mock feed description to token descriptor
+	MockDescriptionToTokenDescriptor = map[string]TokenDescriptor{
+		MockLinkAggregatorDescription: LinkDescriptor,
+		MockWETHAggregatorDescription: WETHDescriptor,
+	}
+)
+
+func DeployFeeds(lggr logger.Logger, chain deployment.Chain) (deployment.AddressBook, map[string]common.Address, error) {
+	ab := deployment.NewMemoryAddressBook()
+	linkTV := deployment.NewTypeAndVersion(PriceFeed, deployment.Version1_0_0)
+	mockLinkFeed, err := deployContract(lggr, chain, ab,
+		func(chain deployment.Chain) ContractDeploy[*aggregator_v3_interface.AggregatorV3Interface] {
+			linkFeed, tx, _, err1 := mock_v3_aggregator_contract.DeployMockV3Aggregator(
+				chain.DeployerKey,
+				chain.Client,
+				DECIMALS,      // decimals
+				MockLinkPrice, // initialAnswer
+			)
+			aggregatorCr, err2 := aggregator_v3_interface.NewAggregatorV3Interface(linkFeed, chain.Client)
```

Comment:
This code is unidiomatic; typically this `NewAggregatorV3Interface` will never return an error unless the ABI is not well formed (which is impossible - guaranteed by the compilation process). Better to check err immediately.

Reply:
We're inside a function that's passed as a parameter and needs to return `ContractDeploy[*aggregator_v3_interface.AggregatorV3Interface]{`. You suggest just return the ContractDeploy twice with different errors after each?

Reply:
hm might be simpler to just have MockFeeds map[TokenSymbol]*mock_aggregator_v3.MockAggregator then when we add real feeds separately have the full OffchainAggregator wrapper

Reply:
We'll give away the flexibility of the tests though with deploying any mock contract by description.